hikari/impl/stateless_cache.py

```diff
@@ -390,3 +391,20 @@ def update_voice_state(
         self, voice_state: voices.VoiceState, /
     ) -> typing.Tuple[typing.Optional[voices.VoiceState], typing.Optional[voices.VoiceState]]:
         raise self._no_cache()
+
```

Comment:
likely a clear messages one for the sake of keeping the interface somewhat consistent as well